despre.html

```diff
@@ -74,13 +74,24 @@
 
       <nav class="main-nav">
         <ul class="main-nav-list">
-          <li><a class="main-nav-link" href="index.html">Home</a></li>
-          <li><a class="main-nav-link" href="about.html">About</a></li>
-          <li><a class="main-nav-link" href="services.html">Services</a></li>
-          <li><a class="main-nav-link" href="blog.html">Blog</a></li>
+          <li>
+            <a class="main-nav-link" href="./index.html">Acasa</a>
+          </li>
+          <li>
+            <a class="main-nav-link main-nav-link-active" href="./despre.html"
+              >Despre</a
+            >
+          </li>
+          <li><a class="main-nav-link" href="./servicii.html">Servicii</a></li>
+          <li><a class="main-nav-link" href="./echipa.html">Echipa</a></li>
+          <li><a class="main-nav-link" href="./galerie.html">Galerie</a></li>
```

Comment:
For each page (Despre, Servicii, Echipa, Fa o programare, etc) we need to create a folder with its own name (despre, servicii, echipa, programare) and place inside each folder it's coresponding HTML file, but rename it to `index.html`.

We do this so that the URL will not include the .html extension at the end and will be more user friendly.

Reply:
This is fixed now